Vagrantfile

```diff
@@ -49,11 +49,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.ssh.forward_x11 = true
 
   # Specific IPs. These is needed because DPDK takes over the NIC.
-  config.vm.network "private_network", ip: "10.100.1.10"
-  config.vm.network "private_network", ip: "10.100.1.11"
+  config.vm.network "private_network", ip: "10.100.1.10", :mac => "020000FFFF00"
+  config.vm.network "private_network", ip: "10.100.1.11", :mac => "020000FFFF01"
 
   # NIC on the same subnet as the two bound to DPDK.
-  config.vm.network "private_network", ip: "10.100.1.255", :mac => "020000FFFFFF"
+  config.vm.network "private_network", ip: "10.100.1.254", :mac => "020000FFFFFF"
```

Comment:
any specific reasoning for this?

Reply:
or just choice.

Reply:
so yeah turns out `10.100.1.255` is the broadcasting address for that `/24` subnet. It probably caused some weird things.